src/plugins/intel_cpu/tools/dump_check/dump_check.py

```diff
@@ -174,7 +174,7 @@ def dump_tensors(core, model, dump_dir = "./cpu_dump", dump_ports="OUT", device_
                 "INFERENCE_NUM_THREADS":1}
     if infer_bf16 == True:
         device_config["INFERENCE_PRECISION_HINT"] = "bf16"
-        device_config["ENFORCE_BF16"] = "YES"
+        # device_config["ENFORCE_BF16"] = "YES"
```

Comment:
Why it commented out now? Maybe worth to delete at all?

Reply:
Applied. Thanks.